docs/MO_DG/prepare_model/MO_Python_API.md

```diff
@@ -0,0 +1,100 @@
+## Model Optimizer Python API {#openvino_docs_MO_DG_Python_API}
+
+Model Optimizer (MO) has a Python API for model conversion, which is represented by the `convert_model()` method in the openvino.tools.mo namespace.
+  `convert_model()` has all the functionality available from the command-line tool, plus the ability to pass Python objects (models, extensions) to `convert_model()` directly from memory without leaving the training environment (Jupyter Notebook or training scripts).
+  `convert_model()` returns an openvino.runtime.Model object which can be compiled and inferred or serialized to IR.
+
+```sh
+from openvino.tools.mo import convert_model
+
+ov_model = convert_model("resnet.onnx")
+```
+
+MO Python API allows the conversion of PyTorch models.
+
+```sh
+import torchvision
+
+model = torchvision.models.resnet50(pretrained=True)
+ov_model = convert_model(model, input_shape=[1,3,100,100])
+```
+
+The following types are supported as an input model for `convert_model()`:
+
+* PyTorch - `torch.nn.Module`, `torch.jit.ScriptModule`, `torch.jit.ScriptFunction`. Refer to the [Converting a PyTorch Model](@ref openvino_docs_MO_DG_prepare_model_convert_model_Convert_Model_From_PyTorch) article for more details.
+* TensorFlow/ TensorFlow2 / Keras - `tf.keras.Model`, `tf.keras.layers.Layer`, `tf.compat.v1.GraphDef`, `tf.Module`, `tf.compat.v1.wrap_function`, `tf.compat.v1.session`, `tf.train.checkpoint`, `tf.python.training.tracking.base.Trackable`(with limitations). Refer to the [Converting a TensorFlow Model](@ref openvino_docs_MO_DG_prepare_model_convert_model_Convert_Model_From_TensorFlow) article for more details.
+
+`convert_model()` accepts all parameters available in the MO command-line tool. Parameters can be specified by Python classes or string analogs, similar to the command-line tool.
+Example 1:
```

Comment:
So, to understand how to use it I need first to learn all existing parameters in Model Optimizer and then apply "conversion from the command line to Python" to learn how to use it in Python. Some time ago we discussed the possibility to provide both command line and Python documentation along all the topics explained in MO section like we have for C++ and Python: via tabs. Probably there is no much time to achieve it, but could you estimate it at least? It would be more aligned with approach "Python first".

Reply:
Great, let's create a ticket for that.

Reply:
Created the ticket: CVS-103302